webui/react/src/pages/WorkspaceDetails.tsx

```diff
@@ -259,16 +266,12 @@ const WorkspaceDetails: React.FC = () => {
         />
       }
       id="workspaceDetails">
-      {rbacEnabled ? (
-        <Pivot
-          activeKey={tabKey}
-          destroyInactiveTabPane
-          items={tabItems}
-          onChange={handleTabChange}
-        />
-      ) : (
-        <WorkspaceProjects id={id} pageRef={pageRef} workspace={workspace} />
-      )}
+      <Pivot
```

Comment:
As [WEB-809], the Pivot should now be visible even if rbac is off.

Reply:
Good question for @ClaireNeveu  or @alexjdetermined. In my mind, the members tab does not make much sense out outside of the context of RBAC.

Reply:
yes - the members tab should be hidden for non RBAC deployments.

Reply:
We do not have a member tab in OSS, that is rbac only.